component.json

```diff
@@ -13,12 +13,65 @@
       }
     },
     "jsonToXml": {
-      "title": "JSON to XML",
-      "main": "./lib/actions/jsonToXml.js",
+      "title": "JSON to XML (Deprecated)",
+      "main": "./lib/actions/jsonToXmlOld.js",
+      "deprecated": true,
       "description": "Takes the body of message passed into the component and converts to generic XML string",
       "metadata": {
         "in": {},
-        "out": "./lib/schemas/jsonToXml.out.json"
+        "out": "./lib/schemas/jsonToXmlOld.out.json"
+      }
+    },
+    "jsonToXmlV2": {
+      "title": "JSON to XML",
+      "main": "./lib/actions/jsonToXml.js",
+      "description": "Takes the result of a JSONata expression and creates corresponding XML as either a string or an attachment",
```

Comment:
There is the new help links section: https://docs.elastic.io/guides/configuring-help-links.html
`description` text is not displaying on the frontend anymore.